app/javascript/css/contrast_level_aa.scss

```diff
@@ -36,3 +38,31 @@
     --enrollment-status-blue: #003260;
     --enrollment-status-grey: #3B3A39;
 }
+
+.warning-notice {
+  padding: 8px 14px 8px 14px;
+  border-radius: 18px;
+  border: 1px;
+  background-color: var(--soft-yellow);
+}
+.warning-icon-small {
+  background-image: url('../../assets/images/warning-icon-small.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  color: transparent !important;
+  padding: 1px;
+  margin-right: 5px !important;
+}
+.warning-icon {
+    background-image: url('../../assets/images/warning-icon.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    color: transparent !important;
+    font-size: 27px !important;
+  }
+.in-progress-icon {
```

Comment:
Can add a reason why we do this here if needed. In short, we don't want to show the alternate icons when we are using this new theming so we turn them transparent.

Reply:
Added note: https://github.com/ideacrew/enroll/pull/3398/commits/beac84742e39cea449a9c117105f55c6597be94c

I wasn't able to find a better way, but if you can think of one please let me know. Didn't want to add any conditional checks in [_status](https://github.com/ideacrew/enroll/pull/3398/files#diff-d9e6ef847a39de598b03fb848b804b6a64bfbd4ee03b77daf3d1e6a4355b2c3fR23) or [info_needed](https://github.com/ideacrew/enroll/pull/3398/files#diff-bbcd5a45b4e20e6e67ab5a86773762f688a612e4b8bcc1e0b4c6cb19ef894639R4)

Reply:
fixed here: https://github.com/ideacrew/enroll/pull/3398/commits/dc15ac6b0625a4f8572df90df49e29349beb3f8d#diff-0f51e756c58b7b7a714c9b46ee136577e97e38a2307b1e57a12f80523995e0baL64-L71